test/hotspot/jtreg/vmTestbase/nsk/jvmti/CompiledMethodUnload/compmethunload001.java

```diff
@@ -47,7 +47,7 @@ public class compmethunload001 {
     private final static String CLS_TO_BE_UNLOADED =
         "nsk.jvmti.CompiledMethodUnload.compmethunload001u";
 
-    private final static int MAX_ITERATIONS = 5;
+    private final static int MAX_ITERATIONS = 50;
```

Comment:
Can you explain the need for this change.

Reply:
The code "wait on a monitor and notify from JNI" seems too complex for me. So I just limit the loop to no more than 10 seconds.